tests/test_image.py

```diff
@@ -97,6 +107,10 @@ def test_ImageReader(data_path):
 
     pipeline.run()
 
+    with Pipeline() as pipeline:
+        image = ImageReader(d, pil_mode="L")
```

Comment:
`pil_mode` should be parametrized: `@pytest.mark.parametrize("pil_mode", [None, "L", "RGB"])`